EXCLUSION.md

```diff
@@ -1,3 +1,41 @@
-TwoFactorAuth.org Blacklist
-=================
+# TwoFactorAuth.org excluded categories and websites
 
+Below is a list of categories and websites that we've opted out from listing on TwoFactorAuth.org. If you want to make a copy (fork) of our site to list any of these sites you're welcome to do so as long as you comply with our [license](https://github.com/2factorauth/twofactorauth/blob/master/LICENSE).
+
+## Categories
+
+*   #### Pornographic sites
+
+    As the volunteers of [2factorauth](https://github.com/2factorauth) need to go through all sites to validate that the site/service provides 2FA to it's users and verify that the pull request meets our style guidelines we've chosen not to list pornographic sites.  
```

Comment:
"Pornographic sites" is ambiguous. Is reddit a ponographic site? It certainly has a lot of pornographic content.
```suggestion
    As the volunteers of [2factorauth](https://github.com/2factorauth) need to go through all sites to validate that the site/service provides 2FA to it's users and verify that the pull request meets our style guidelines we've chosen not to list sites that primarily host and serve pornographic content.  
```

Reply:
Resolved with 3024fb2